src/server/server.zig

```diff
@@ -322,6 +355,12 @@ fn init(name: []const u8, singlePlayerPort: ?u16) void { // MARK: init()
 		std.log.err("Failed to create world: {s}", .{@errorName(err)});
 		@panic("Can't create world.");
 	};
+
+	const settingsZon = world.?.wio.dir.readToZon(main.stackAllocator, "server.zig.zon") catch .null;
```

Comment:
I don't think this is the right place. We shouldn't store IP addresses anywhere in the world folder, since that is something that is naturally shared with others.

Plus it would be weird to carry over IP bans from adventure maps shared online.